docs/tutorial_GPU.ipynb

```diff
@@ -0,0 +1,5769 @@
+{
```

Comment:
<p>Can we explain more explicitly why we are filtering <code>SST * area</code>  rather than just <code>SST</code>.  This could be very confusing to new users.</p><p> </p><p>(Side note: it seems like this should be handled by the kernel, not by the user.)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ocean-eddy-cpt/gcm-filters/pull/69/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>